coordinator/tributary/src/tendermint/mod.rs

```diff
@@ -343,6 +348,7 @@ impl<D: Db, T: TransactionTrait, P: P2p> Network for TendermintNetwork<D, T, P>
 
     // add tx to blockchain and broadcast to peers
     // TODO: Make a function out of this following block
+    // Why? Seems like a single caller.
```

Comment:
If it's still a single caller and doesn't appear to have unique value, then likely an old comment we can remove :+1: